### PR TITLE
[doc] PR for Update/Rewrite the "Create a multi-region universe" page

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-3342

Preview: https://deploy-preview-11734--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/create-deployments/create-universe-multi-region/

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-region.md [hunk 1]
```diff
@@ -113,28 +101,13 @@ $ java -jar /home/yugabyte/tserver/java/yb-sample-apps.jar \
             --with_local_dc <REGION>
 ```
 
-You can find the region codes for each of the nodes by browsing to the **Nodes** tab for this universe in the Yugabyte Platform console. A screenshot is shown below. In this example, the value for `<REGION>` is:
+You can find the region code of each node by navigating to **Nodes** and looking under the **CLOUD INFO** column: the first part before the slash indicates the region. For example, `us-west1`.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-region.md:104]: worth mentioning this would be back in platform UI?

**Review by ddhodge — APPROVED** [review]
LGTM

File: docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-region.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-region.md:104]: I think it's okay as is

**Comment by lizayugabyte** [comment]
@VijiYB Please let me know if you or someone in your team will be able to review this doc PR. If not, then I'll merge it as is and then we'd deal with inaccuracies, if any, via a new PR.

**Pull request merged**